packages/sdk/src/reads/getAssetAmounts.ts

```diff
@@ -0,0 +1,29 @@
+import type { Address, PublicClient } from "viem";
+import type { ReadContractParameters } from "../utils/viem.js";
+import { getAssetAmount } from "./getAssetAmount.js";
+
+export async function getAssetAmounts(
+  client: PublicClient,
+  args: ReadContractParameters<{
+    owner: Address;
+    assets: Address[];
+  }>,
+) {
+  const balances = await Promise.all(
+    args.assets.map(async (asset) => {
+      const amount = await getAssetAmount(client, {
+        ...args,
```

Comment:
@KedziaPawel Note how I'm forwarding the passed `args` (so the user can pass in a block number, etc)